webcompat/static/js/lib/bugform-validation.js

```diff
@@ -0,0 +1,48 @@
+/* exported Validation */
+function Validation() {
+  var GITHUB_REGEXP = /^[a-z\d](?:[a-z\d]|-(?=[a-z\d])){0,38}$/i;
+  var ALLOWED_IMG_FORMAT = ["jpg", "jpeg", "jpe", "png", "gif", "bmp"];
+
+  var isReportableURL = function(url) {
```

Comment:
Note: I guess this validation kinda sucks, since it will accept just `http:blah`. But we can improve that in a follow-up bug. Historically we haven't wanted to be *too* strict, but we can probably improve things here.

Reply:
yeah I've noticed that :) wasn't sure whether I should change it to be more strict. I'll create a bug for that